WordPress/Classes/ViewRelated/What's New/WhatIsNewViewController.swift

```diff
@@ -0,0 +1,66 @@
+
+import UIKit
+
+
+class WhatIsNewViewController: UIViewController {
```

Comment:
For example, you have a great summary in the PR that could go here:

```suggestion
/// WhatIsNewViewController: adds the "What's New" Entry in app settings and presents a modal that will become the Feature Announcement screen
///
class WhatIsNewViewController: UIViewController {
```

Reply:
Absolutely right! I'll add some description to both `WhatIsNewViewController` and `WhatIsNewView`